operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -30,6 +30,8 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## 4.2.0
 
+* [FEATURE] Add cache support for GEM's admin bucket. #3740
```

Comment:
Should go to main/unreleased

Reply:
Also since the changelog is the primary source of information for upgrade, let's mention that it will be enabled by default in the small, large values files.

Reply:
> Should go to main/unreleased
Sorry, I keep forgotten whether to put changelog in the upcoming release or in main/unreleased. 

> Also since the changelog is the primary source of information for upgrade, let's mention that it will be enabled by default in the small, large values files.
Added the comment